src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: flex;
+  justify-content: space-between;
+  min-height: 216px;
+  padding: $padding-page-l 0;
+}
+
+.SecondaryHero-message {
+  font-size: 16px;
+  line-height: 1.5;
+  margin-top: 16px;
+  width: 25%;
+}
+
+.SecondaryHero-message-headline {
+  font-size: $font-size-l;
+  line-height: 1.333;
+}
+
+.SecondaryHero-message-description {
+  padding: $padding-page 0;
+}
+
+.SecondaryHero-modules {
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(3, 1fr);
+  width: 70%;
+}
+
+.SecondaryHero-module {
+  align-items: center;
+  background-color: $white;
+  border-radius: $border-radius-s;
+  display: flex;
+  flex-direction: column;
+  // flex-grow: 0;
+  font-size: 18px;
```

Comment:
Could this drop down to `$font-size-m` (16px) or is that too small?

Reply:
It could. The mock has this set at `18`, which is why I made it that. The mock shows that this is slightly larger than the font used for the description for the left-side message (`SecondaryHero-message`), which is why I used `18`. I feel like we're starting to play a bit fast and loose with the mocks the way we're changing a lot of the sizes, but I think it's probably okay as long as we keep them relatively the same. To change this so that it's the same size as another element in the mock, which the mock defines as being smaller than this, seems like maybe going too far.

Reply:
OK. I suggest declaring a new font variable for `18px` just so we have all the sizes in the same place. It could be something super specific like `$font-size-hero-description` or whatever.

Reply:
Interestingly, there is a `$font-size-m` defined as `18px` in `ui/css/vars.scss`, but I guess those values aren't used for components in the `amo` folder. I'll just create a specific one for now, as you suggest.